vmdb/app/models/ems_refresh/refreshers/foreman_refresher.rb

```diff
@@ -0,0 +1,13 @@
+require 'manageiq_foreman/inventory'
+
+module EmsRefresh
+  module Refreshers
+    class ForemanRefresher
+      def self.refresh(providers)
+        providers.each do |provider|
+          EmsRefresh.refresh(provider.provisioning_manager)
```

Comment:
```
foreman(provider) -> foreman_provisioning_manager -> foreman_configuration_manager
```

Since most of the time we want to update the `configuration_manager` and not the `provisioning_manager`, refreshing the `configuration_manager` instead may make more sense.
The first case would be a little slower, but the expectations are that it will be a little slower, so that may be fine.